components/developer/rust/patches/vendor_backtrace-sys_src_libbacktrace_configure.patch

```diff
@@ -0,0 +1,16 @@
+$NetBSD: patch-vendor_backtrace-sys_src_libbacktrace_configure,v 1.1 2019/03/03 09:16:21 he Exp $
+
+Use pkgsrc libtool.
```

Comment:
s/pkgsrc/system/ ?

Reply:
Well I could but keeping that comment to be exact to the patch makes diff with pkgsrc easier. I expect patches to be added and removed a lot between releases until rust on illumos is stable enough for them to not be needed anymore. And I am really happy to let others to that porting work :)